import/webview/WebView.qml

```diff
@@ -15,12 +16,17 @@ import Sailfish.Silica.private 1.0 as SilicaPrivate
 import Sailfish.WebView 1.0
 import Sailfish.WebView.Popups 1.0
 import Sailfish.WebView.Pickers 1.0
+import Sailfish.WebView.TextSelection 1.0
```

Comment:
As said elsewhere maybe SailfishWebView.Controls import for the TextSelection and friends.

Reply:
Let's handle merging of Popups and Pickers to the Controls later.